RecoTracker/MkFit/plugins/MkFitIterationConfigESProducer.cc

```diff
@@ -110,37 +93,41 @@ namespace {
                                         const TrackVec &in_seeds,
                                         const EventOfHits &eoh,
                                         IterationSeedPartition &part) {
-    // Seeds are placed into eta regions and sorted on region + eta.
+    // Define first (mkFit) layer IDs for each strip subdetector.
+    const int tib1_id = 4;
+    const int tob1_id = 10;
+    const int tidp1_id = 21;
+    const int tidn1_id = 48;
+    const int tecp1_id = 27;
+    const int tecn1_id = 54;
```

Comment:
Could be `constexpr`.

Reply:
Switched to constexpr, since it should make no difference.